docs/CustomCapability.md

```diff
@@ -1 +1,104 @@
+We'll also map our two custom functions we've just created and assign them to the methods the source code is expecting them to be.
+
+```javascript
+// require in the Asymmetrik FHIR Server
+const FHIRServer = require('@asymmetrik/node-fhir-server-core');
+
+/**
+ * Generates a custom Capability Statement
+ * @param {object} resources - The list of the supported resources. This is taken from supported Profiles declared in the FHIR Server config.
+ */
+let customCapabilityStatement = resources => {
+  let CapabilityStatement = require(FHIRServer.resolveSchema(base_version, 'CapabilityStatement'));
+
+  return new CapabilityStatement({
+    status: 'draft',
+    date: '20190418',
+    publisher: 'The Publisher',
+    kind: 'instance',
+    software: {
+      name: 'My super cool Software',
+      version: '1.0.0',
+      releaseDate: '20190418'
+    },
+    implementation: {
+      description: 'Your FHIR Server',
+      url: 'http://yourfhirserver.com/'
+    },
+    fhirVersion: base_version.replace(/_/g, '.'),
+    acceptUnknown: 'extensions',
+    format: ['application/fhir+json'],
+    rest: resources
+  });
+};
+
+/**
+ * Generates a custom Security Statement
+ * @param {object} securityUrls - This is apart of the Capability Statement. It provides information about the Security of the current implementation.
+ * If provided, this will be derived from the information you pass to the 'Security' property from the FHIR Server Config.
+ * The below Security Statement is returned by default. This is probably only a place holder for now.
+ */
+let customSecurityStatement = securityUrls => {
+  return {
+    cors: true,
+    service: [
+      {
+        coding: [
+          {
+            system: 'http://hl7.org/fhir/restful-security-service',
+            code: 'SMART-on-FHIR'
+          },
+        ],
+        text: 'Custom OAuth2 using SMART-on-FHIR profile (see http://docs.smarthealthit.org)'
+      },
+    ],
+    extension: [
+      {
+        url: 'http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris',
+        extension: securityUrls
+      }
+    ]
+  };
+};
+
+/**
+ * Exports Custom functions to be used in Server Configuration
+ * @param {object} args - This will pass us the FHIR Spec Version of the Capability Statement. This comes from the route you're hitting.
+ */
+module.exports.generateStatements = (args) => {
+  base_version = args.base_version;
+  return {
+    makeStatement: customCapabilityStatement,
+    securityStatement: customSecurityStatement
+  };
+};
+```
+
+### Using the Statement Generator in the Config
+Now that we've built out our Statement Generator and have customized the Capability Statement info to our liking, we can add it to our configuration by doing the following: 
+```javascript
```

Comment:
Just to be thorough and prevent any potential confusion, can we make this a full example and just add the following into it:

```javascript
const FHIRServer = require('@asymmetrik/node-fhir-server-core');

let config = {}; // Use the config and other things you provided here
let server = FHIRServer.initialize(config);
...
```